horizons/__init__.py

```diff
@@ -0,0 +1,307 @@
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+import dateutil.parser
+from common.minimodel import Target, TargetTag, Site
+from common.helpers.helpers import dms2rad, hms2rad
+import numpy as np
+import numpy.typing as npt
+import requests
+from typing import Tuple, Union, List
+import logging
+import contextlib
+import os
+
+
+@dataclass
+class Angle:
+    """
+    Angle in radians.
+    """
+    µasPerDegree: float = 60 * 60 * 1000 * 1000
+
+    @staticmethod
+    def to_signed_microarcseconds(angle: float) -> float:
+        """
+        Convert an angle in radians to a signed microarcsecond angle.
+        """
+        degrees = Angle.to_degrees(angle)
+        if degrees > 180:
+            degrees -= 360
+        return degrees * Angle.µasPerDegree
+ 
+    def to_degrees(angle: float) -> float:
+        """
+        Convert an angle in radians to a signed degree angle.
+        """
+        return angle * 180.0 / np.pi
+
+    @staticmethod
+    def to_microarcseconds(angle: float) -> float:
+        """
+        Convert an angle in radians to a signed microarcsecond angle.
+        """
+        return Angle.to_degrees(angle) * Angle.µasPerDegree
```

Comment:
Yes... this is probably what is causing the issues, I would think... major loss of precision here over GPP's representation of angles, which would cause our test cases to fail since we're using `float` and they are using `Long` and storing everything directly (I believe) in microarcseconds whereas we are storing in signed decimal degrees, which is much less precise.